### PR TITLE
Set up Travis CI for testing.

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

**Comment by antiagainst** [comment]
@google/shaderc please review :)

**Comment by AWoloszyn** [comment]
+1

File: cmake/utils.cmake [hunk 1]
```diff
@@ -128,11 +128,11 @@ function(combine_static_lib new_target target)
   string(REPLACE ";" "\\\\naddlib " all_libs_string "${all_libs}")
 
   set(ar_script
-    "create lib${new_target}.a\\\\naddlib ${all_libs_string}\\\\nsave\\\\n end")
+    "create lib${new_target}.a\\\\naddlib ${all_libs_string}\\\\nsave\\\\nend")
 
   add_custom_command(OUTPUT lib${new_target}.a
     DEPENDS ${all_libs}
-    COMMAND echo -e ${ar_script} | ${CMAKE_AR} -M)
+    COMMAND ${ECHO_EXE} -e ${ar_script} | ${CMAKE_AR} -M)
```
Comment (dmircevski) [inline cmake/utils.cmake]: How is this related to Travis?

Comment (antiagainst) [inline cmake/utils.cmake]: Travis CI uses Ubuntu 12.04. it's echo setup (from bash builtin) doesn't support the `-e` option.

**Comment by dmircevski** [comment]
+2
